### DISCUSSION TITLE
Text get cut off on terminal.

### DISCUSSION THREAD

**Discussion by AyaanZaveri** [coder/code-server#4292] [Q&A] [answered]
![Screen Shot 2021-10-01 at 3 37 08 PM](https://user-images.githubusercontent.com/63752541/135677089-45cefbc6-9175-478f-bbca-216e5eedc7c9.png)

On Version _3.11.0_ whenever this happened I would refresh the page and it would be fixed, but on _3.12.0_ it doesn't seem to fix. Also, on Safari the terminal text is invisible, which never occurred on _3.11.0_. 

**Version:** 3.12.0
**Browser:** Brave (Chromium Based)
**Device:** MacBook Air M1

**Comment by jsjoeio** [comment]
Hmm...do you know if it can be reproduced in https://xtermjs.org/? 

Are you using any custom fonts or anything like that?

**Reply by AyaanZaveri** [reply]
This is what the Xterm website looks like:
**Brave:**
<img width="867" alt="Screen Shot 2021-10-01 at 3 59 31 PM" src="https://user-images.githubusercontent.com/63752541/135679519-55aa2af7-a88a-4c63-b938-2f2afb2bf6b4.png">
On Brave some of the text was cut off.

**Safari:**
<img width="812" alt="Screen Shot 2021-10-01 at 3 59 39 PM" src="https://user-images.githubusercontent.com/63752541/135679647-4e4bbbce-6c76-4e72-9c40-856f615288cf.png">
On Safari it was invisible.

**Code-Server Terminal Font** (Cut Off): Menlo, Monaco, 'Courier New', monospace
**Code-Server Editor Font** (Looks Perfect): Menlo, Monaco, 'Courier New', monospace

**Reply by jsjoeio** [reply]
> On Brave some of the text was cut off.

Hmm...that makes me think this could be upstream. Do you agree? I would look into opening an [issue upstream](https://github.com/xtermjs/xterm.js/issues)

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Hey, won't it be Brave's fingerprinting system for protection? Why not turn that off and try again? @jsjoeio told me that Brave had a fingerprinting wall for security and disabling it worked, so does it work now?

**Reply by AyaanZaveri** [reply]
Thanks so much! That worked.
